scss/typography/_dropcaps.scss

```diff
@@ -0,0 +1,64 @@
+////
+/// @group typography-dropcaps
+////
+
+/// Number of lines against the rest of the content.
+/// @type Number
+$dropcaps-lines: 3 !default;
+
+/// Line height for the rest of the content.
+/// @type Number
+$dropcaps-line-height: 1.6 !default;
+
+/// If `true`, makes dropcaps uppercase.
+/// @type Boolean
+$dropcaps-uppercase: true !default;
+
+/// Dropcaps component
+/// @param { Number } $lines [$dropcaps-lines] Number of lines against the rest of the content.
+/// @param { Number } $line-height [$dropcaps-line-height] Line height for the rest of the content.
+/// @param { Boolean } $uppercase [$dropcaps-uppercase] If `true`, makes dropcaps uppercase.
+@mixin dropcaps(
+  $lines: $dropcaps-lines,
+  $line-height: $dropcaps-line-height,
+  $uppercase: $dropcaps-uppercase
+) {
+  // Default margin right (Arial/Helvetica) for having a tiny space between them through the right,
+  // See: http://code.stephenmorley.org/html-and-css/creating-drop-caps
+  $-margin-right: 0.0962em;
```

Comment:
@IamManchanda Agreed. These values can "match" with the most common fonts, but these are still values for `Helvetica Neue` and there is no guarantee the render will be still good with other fonts.